Orange/widgets/visualize/owscatterplot.py

```diff
@@ -87,13 +87,16 @@ def score_heuristic(self):
         X = self.master.graph.jittered_data.T
         Y = self.master.data.Y
         mdomain = self.master.data.domain
+        color_domain = mdomain[self.master.graph.attr_color]
```

Comment:
This is misleading - don't call a variable a "domain". Rename `color_domain` to `color_var` or something.